src/app/components/search/search-form.html

```diff
@@ -45,3 +56,26 @@
   </form>
 
 </div>
+
+<!--
```

Comment:
@rupl Would it make sense to completely remove this, since it seems commented out ? Or are you keeping it in there for a specific reason ?

Reply:
yes totally.

Since these migrations took a long time I had worked with both copies next to each other for some time. If you see any other commented blocks of HTML, please do call them out and I'll remove all of them.